[2024-07-22T02:36:11.847+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.sum_it scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-07-22T02:36:11.853+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.sum_it scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2024-07-22T02:36:11.854+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:11.854+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:36:11.854+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:11.862+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): sum_it> on 2023-01-02 00:00:00+00:00
[2024-07-22T02:36:11.865+0000] {standard_task_runner.py:55} INFO - Started process 3371 to run task
[2024-07-22T02:36:11.867+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'sum_it', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '1247', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmp28wxrro2']
[2024-07-22T02:36:11.868+0000] {standard_task_runner.py:83} INFO - Job 1247: Subtask sum_it
[2024-07-22T02:36:11.909+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1247 for task sum_it ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2023-01-02T00:00:00+00:00/task_id=sum_it/attempt=1.log'; 3371)
[2024-07-22T02:36:11.927+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:36:11.938+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
